variables.tf

```diff
@@ -104,6 +104,20 @@ variable "engine_version" {
   # http://docs.aws.amazon.com/cli/latest/reference/rds/create-db-instance.html
 }
 
+variable "major_engine_version" {
+  type        = "string"
+  description = "Database MAJOR engine version, depends on engine type"
+  default     = ""
```

Comment:
will it work when `default` is an empty string?
If not, we need to remove `default` and make it a required param.

Reply:
Good catch, this has to be provided.  The docs were already updated but I didn't remove the default value.   I'll re-submit the PR

Reply:
just remove the `default` and then rebuild README by executing these commands:

```
make init
make readme/deps
make readme
```

Reply:
Yep - just did that (forgot that the PR will just update from my branch)